packages/p2p/src/components/orders/order-table-row.jsx

```diff
@@ -50,6 +54,10 @@ const OrderRowComponent = observer(({ data: order, style }) => {
             setOrderState(createExtendedOrderDetails(order.order_details, client.loginid, props.server_time));
             clearInterval(interval);
         } else {
+            const { is_active_tab } = general_store;
+            if (is_active_tab) {
+                setIsTimerVisible(true);
+            }
```

Comment:
For one variable, it's okay to use directly from the store. For more than one, we can deconstruct. What do you think?

```suggestion
            if (general_store.is_active_tab) {
                setIsTimerVisible(true);
            }
```